client/jetpack-connect/woo-dna-config.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from 'config';
+
+export default isEnabled( 'jetpack/connect/woo-dna' )
+	? {
+			'woocommerce-payments': {
+				name: ( translate ) => translate( 'WooCommerce Payments' ),
+				helpUrl: 'https://docs.woocommerce.com/document/payments/', // TODO: Write a WCPay-specific page for Jetpack connection troubles
```

Comment:
@allendav FYI, we should have this page ready before releasing WCPay 1.1 (which will include the new connection flow).

Reply:
OK - at that URL, right?  Please open an issue (type: task), start a google doc where we can collaborate on the content, and add the issue to the sprint 14 pre-planning post

Reply:
No, that URL will need to be changed, right now it points to our WCPay docs. It will need to be a different page or an "anchored" section on that page. I've created https://github.com/Automattic/woocommerce-payments/issues/708 and opened a doc to collaborate on this.